docs/source/index.md

```diff
@@ -5,8 +5,8 @@ project that covers many different software offerings and tools, including the
 popular [Jupyter Notebook](https://jupyter-notebook.readthedocs.io/en/stable/)
 and [JuyterLab](https://jupyterlab.readthedocs.io/en/stable/) web-based
 notebook authoring and editing applications. The Jupyter project and its
-subprojects all center around providing tools for interactive computing with
-computational notebooks.
+subprojects all center around providing tools (and [standards](https://jupyter-client.readthedocs.io/en/stable/messaging.html))
```

Comment:
there is more to standards than messaging, https://jupyter.org/ summarises it well:

![Screenshot from 2023-05-24 20-52-50](https://github.com/jupyter/jupyter/assets/5832902/f77dec58-4818-4bda-a65a-9bba2e020f98)

maybe a better page to link to would be https://docs.jupyter.org/en/latest/#sub-project-documentation
which has this neat table:

![Screenshot from 2023-05-24 20-54-01](https://github.com/jupyter/jupyter/assets/5832902/919375fe-6c30-4f3f-a764-bd870aa2d649)